packages/rollup-plugin-html/src/RollupPluginHTMLOptions.ts

```diff
@@ -44,6 +44,17 @@ export interface GeneratedBundle {
   bundle: OutputBundle;
 }
 
+export interface Attribute {
```

Comment:
maybe we can use https://developer.mozilla.org/en-US/docs/Web/API/Attr here?

Reply:
there is also `import { Attribute } from '@web/parse5-utils';` with

```js
/**
 * Element attribute.
 */
export interface Attribute {
    /**
     * The name of the attribute.
     */
    name: string;

    /**
     * The value of the attribute.
     */
    value: string;

    /**
     * The namespace of the attribute.
     */
    namespace?: string;

    /**
     * The namespace-related prefix of the attribute.
     */
    prefix?: string;
}
```

will use that for now